src/components/memorygame/changeCardStatePropertyToOpposite.jsx

```diff
@@ -0,0 +1,48 @@
+// manipulates isFlipped and isMatched property of card a state
+// property example : "isMatched"
+// cardStates is an array of objects(states)
+// cardStates example :
+// [
+//   {
+//      cardKey:"",
+//      img:{src:"",imgKey:""},
+//      isFlipped:false,
+//      isMatched:false,
+//      description:"",
+//      link:"",
+//      phrase:""
+//   },
+// ...]
+// targetCard is a state obj.
+// Object in line 6-14 is an example for targetCard
+const changeCardStatePropertyToOpposite = (
+  property,
```

Comment:
Where is the property prop coming from? I got a bit confused with the if conditions

Reply:
Since this is a function, property is not a prop. It is a parameter. Name is misleading but function changes property of a state.So, property stands for target state property. I added a comment about where to see the use of function. Also, I have changed property into stateProperty. I'm not sure if it is descriptive enough. Please let me know

PTAL